app/utils/date-formatting.js

```diff
@@ -29,7 +29,10 @@ function parseDateString(value, offset) {
     // in UTC timezone
     moment.tz.setDefault(offset);
 
-    return value ? moment(verifyTimeStamp(value), parseDateFormats, true) : undefined;
```

Comment:
Would you be better off here making sure that `post.publishedAtOffset` returns `null` when `publishedAt` is null? 

It seems to me the difference between `publishedAt` and `publishedAtOffset` behaviour is now leaking to external components such as this date formatting util which has now also lost the ability to display nothing for empty values.